src/hotspot/share/gc/shared/parallelCleaning.cpp

```diff
@@ -49,15 +49,15 @@ CodeCacheUnloadingTask::~CodeCacheUnloadingTask() {
   CodeCache::verify_clean_inline_caches();
 }
 
-void CodeCacheUnloadingTask::claim_nmethods(CompiledMethod** claimed_nmethods, int *num_claimed_nmethods) {
-  CompiledMethod* first;
-  CompiledMethodIterator last(CompiledMethodIterator::all_blobs);
+void CodeCacheUnloadingTask::claim_nmethods(nmethod** claimed_nmethods, int *num_claimed_nmethods) {
+  nmethod* first;
+  NMethodIterator last(NMethodIterator::all_blobs);
```

Comment:
FWIW, `all_blobs` is slightly confusing name when nmethods are a subset of all "code blobs". We might want to consider renaming this to `NMethodIterator::all` (in a separate RFE).